examples/3d/3d_scene.rs

```diff
@@ -28,8 +28,14 @@ fn setup(
         ..Default::default()
     });
     // light
-    commands.spawn_bundle(PointLightBundle {
-        transform: Transform::from_xyz(4.0, 8.0, 4.0),
+    let transform = Transform::from_xyz(4.0, 8.0, 4.0).looking_at(Vec3::ZERO, Vec3::Y);
+    let mut directional_light = DirectionalLight::default();
+    directional_light.illuminance = 10_000.0;
+    directional_light.set_direction(transform.forward());
+
+    commands.spawn_bundle(DirectionalLightBundle {
+        transform,
+        directional_light,
```

Comment:
```suggestion
        directional_light
        transform,
```

Per CI, these needs to be swapped.